pages/c/_cluster/apps/charts/install.vue

```diff
@@ -186,6 +186,12 @@ export default {
     this.updateStepOneReady();
 
     this.preFormYamlOption = this.valuesComponent || this.hasQuestions ? VALUES_STATE.FORM : VALUES_STATE.YAML;
+
+    // Look for annotation to say this app is a legacy migrated app (we look in either place for now)
+    if ((this.existing.metadata?.annotations?.[CATALOG_ANNOTATIONS.MIGRATED] === 'true') ||
```

Comment:
`existing` doesn't exist for new installs

Reply:
@richard-cox Fixed. Also simplified the check, based on the reply in the issue